clients/web/src/init.js

```diff
@@ -17,8 +17,10 @@ _.extend(girder, {
     models: {},
     collections: {},
     views: {},
-    apiRoot: $('#g-global-info-apiroot').text(),
-    staticRoot: $('#g-global-info-staticroot').text(),
+    apiRoot: $('#g-global-info-apiroot').text().replace(
+        'TESTURL', 'http://' + window.location.host),
+    staticRoot: $('#g-global-info-staticroot').text().replace(
+        'TESTURL', 'http://' + window.location.host),
```

Comment:
If we must do this replacement, I think we should indicate that this is a variable, and also decouple it from the notion of testing since it will affect non-testing environments as well. Something like `%HOST%` or `$HOST`.